spec/rubocop/cop/style/block_delimiters_spec.rb

```diff
@@ -338,21 +408,6 @@
       RUBY
     end
 
-    it 'autocorrects do-end to {} if it is a functional block' do
-      expect_offense(<<~RUBY)
-        foo = map do |x|
-                  ^^ Prefer `{...}` over `do...end` for functional blocks.
-          x
-        end
-      RUBY
-
-      expect_correction(<<~RUBY)
-        foo = map { |x|
-          x
-        }
-      RUBY
-    end
```

Comment:
The tests that were starting on lines 341 and 371 were duplicates and have been removed now that the tests higher up have `expect_correction` checks.